src/core/pointcloud/qgspointcloudeditingindex.cpp

```diff
@@ -44,6 +44,7 @@ QgsPointCloudEditingIndex::QgsPointCloudEditingIndex( QgsPointCloudLayer *layer
   mZMax = mIndex.zMax();
   mRootBounds = mIndex.rootNodeBounds();
   mSpan = mIndex.span();
+  mFilterExpression = mIndex.subsetString();
```

Comment:
what if the filter expression changes while we're in editing mode - is this going to be refreshed? I would prefer not to have mFilterExpression here and rather get it when needed from mIndex